assume/common/units_operator.py

```diff
@@ -395,8 +395,13 @@ def write_learning_to_output(self, start: datetime, marketconfig: MarketConfig):
 
                 output_agent_list.append(output_dict)
 
-        db_aid = self.context.data_dict.get("learning_output_agent_id")
-        db_addr = self.context.data_dict.get("learning_output_agent_addr")
+        db_aid = self.context.data_dict.get(
+            "learning_output_agent_id"
+        ) or self.context.data_dict.get("output_agent_id")
+        db_addr = self.context.data_dict.get(
+            "learning_output_agent_addr"
+        ) or self.context.data_dict.get("output_agent_id")
+
```

Comment:
why is this needed?
it must be output_agent_addr at least..?

Reply:
if it is only output agent, we don't write the RL params at all. It would be still a good idea to write them (for example during evaluations). So this is always written

Reply:
oh the address, haha, saw it, thanks

Reply:
But then you should just add the learning_output_agent config on validation runs too - instead of changing this here?